nixos/tests/web-apps/mastodon/mastodon-std.nix

```diff
@@ -9,8 +9,8 @@ let
   '';
 
   hosts = ''
-    192.168.2.10 ca.local
-    192.168.2.11 mastodon.local
```

Comment:
Are the ips of any particular importance? If not, why not keep the old ones and reuse them in the other test?

Reply:
It is easier to navigate the configuration by IP-addresses:
192.168.2.10x - databases and nginx
192.168.2.20x - mastodon service and client